README.adoc

```diff
@@ -730,6 +730,34 @@ aks_<cluster>_<release>_upgrade_fusion.sh
 
 If you're not running on managed K8s platform such as GKE, AKS, or EKS, you can use Helm to install the Fusion chart to an existing Kubernetes cluster.
 
+For eample, you can run local https://kind.sigs.k8s.io/docs/user/quick-start/[`kind`^] k8s cluster on mac with the following commands.
+
+(To install Fusion locally, you should have a node with at least 12GB of memory, 100GB of disk and 4 CPU cores. The recommended setup is 16GB (or more) of memory and 8+ CPU cores.)
+```
+# assume you have brew installed, if not, please refer to https://brew.sh
+brew install kind
+kind create cluster
+```
+
+Then run the https://github.com/lucidworks/fusion-cloud-native/blob/master/setup_f5_k8s.sh.sh[`setup_f5_k8s.sh`^] script to install Fusion 5.x into local k8s cluster.
+```
```

Comment:
Your comment mentions `setup_f5_k8s.sh` but then your example references `setup_f5_gke.sh`???

Reply:
my bad, it should be `setup_f5_k8s.sh`